code/lib/preview-web/src/render/Render.ts

```diff
@@ -1,4 +1,4 @@
-import type { StoryId, AnyFramework } from '@storybook/types';
+import type { StoryId, Framework } from '@storybook/types';
```

Comment:
Should preview-web (being a WEB package) not import WebFramework?

Or does this refactor sort of mean that this package is really "web"-specific anymore?

Reply:
Correct, I think it should be called `PreviewWithSelection` or something less of a mouthful, rather than `PreviewWeb` now.

Reply:
We could also make a `PreviewWeb` which is defined as `PreviewWithSelection({ view: new WebView, selectionStore: new UrlStore })` and import that in the various places it is used if that's clearer. WDYT @ndelangen? I think it would help me with this typing issue too: https://github.com/storybookjs/storybook/pull/19623/files/b00748bff30107b589409c29dbe79fcd0db2ba54#diff-393e8d3b6eccda147d3a872fd5a9a5132c558a6345969adb34e38e656da37f44